supplementary_style_guide/glossary_terms_conventions/general_conventions/g.adoc

```diff
@@ -46,7 +46,7 @@
 [discrete]
 [[gather]]
 ==== image:images/yes.png[yes] gather (verb)
-*Description*: In Red Hat Virtualization, use "gather" when discussing metrics. Do not use "collect", which is reserved for discussing the Red Hat Virtualization log collector (`ovirt-log-collector`). See the comments in link:https://bugzilla.redhat.com/show_bug.cgi?id=1418659[BZ#1418659 Add fluentd configuration for parsing engine.log] for the discussion regarding this decision.
+*Description*: In Red Hat Virtualization, use _gather_ when discussing metrics. Do not use _collect_, which is reserved for discussing the Red Hat Virtualization log collector (`ovirt-log-collector`). See the comments in link:https://bugzilla.redhat.com/show_bug.cgi?id=1418659[BZ#1418659 Add fluentd configuration for parsing engine.log] for the discussion regarding this decision.
```

Comment:
Quotation marks should be used on first instances of "gather" because it's informing about how to write the term. Same with "collect".